File: src/CCVTAC.Console/PostProcessing/ImageProcessor.cs
using CCVTAC.Console.ExternalTools;

namespace CCVTAC.Console.PostProcessing;

internal static class ImageProcessor
{
    internal static readonly string ProgramName = "mogrify";

    internal static void Run(string workingDirectory, Printer printer)
    {
        ToolSettings imageEditToolSettings = new($"{ProgramName} -trim -fuzz 10% *.jpg", workingDirectory);

        Runner.Run(imageEditToolSettings, [], printer);
    }
}

File: src/CCVTAC.Main/PostProcessing/ImageProcessor.fs
namespace CCVTAC.Main.PostProcessing

open CCVTAC.Main.ExternalTools

module ImageProcessor =

    let private programName = "mogrify"

    let run printer workingDirectory : unit =
        let toolSettings = workingDirectory |> ToolSettings.create $"{programName} -trim -fuzz 10%% *.jpg"
        Runner.runTool printer toolSettings [] |> ignore
